apps/blockchain/starknet/src/tests/bridge_t.cairo

```diff
@@ -694,81 +694,94 @@ mod tests {
         let collection1 = starknet::contract_address_const::<'collection1'>();
         let collection2 = starknet::contract_address_const::<'collection2'>();
         let collection3 = starknet::contract_address_const::<'collection3'>();
-
+        let collection4 = starknet::contract_address_const::<'collection4'>();
+        let collection5 = starknet::contract_address_const::<'collection5'>();
+  
+        let mut spy = spy_events(SpyOn::One(bridge_address));
```

Comment:
Could you please add also event checking for at least 1 call

Reply:
Yes I've added this.